data/assets/scene/solarsystem/planets/earth/magnetosphere.asset

```diff
@@ -0,0 +1,67 @@
+local assetHelper = asset.require('util/asset_helper')
```

Comment:
I assume this file made it in by accident? :D

Reply:
Oops! Removed it.